src/Libraries/RevitNodesUI/Properties/Resources.resx

```diff
@@ -349,4 +349,10 @@
   <data name="FilterTypeSelectorDescription" xml:space="preserve">
     <value>Filter Types for Revit Filters</value>
   </data>
+  <data name="ExportColumnHeadersDescription" xml:space="preserve">
```

Comment:
ExportColumnHeadersDescription is only needed for UI dropdown nodes so it should not be in the Revit Nodes resex. Right?